ykrt/src/compile/jitc_yk/jit_ir.rs

```diff
@@ -939,22 +793,16 @@ pub(crate) enum Type {
     Unimplemented(String),
 }
 
-impl JitIRDisplay for Type {
-    fn to_string_impl(
-        &self,
-        m: &Module,
-        s: &mut String,
-        nums: &LocalNumbers<'_>,
-    ) -> Result<(), Box<dyn Error>> {
+impl fmt::Display for Type {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
         match self {
-            Self::Void => s.push_str("void"),
-            Self::Integer(it) => it.to_string_impl(m, s, nums)?,
-            Self::Ptr => s.push_str("ptr"),
-            Self::Func(ft) => ft.to_string_impl(m, s, nums)?,
+            Self::Void => write!(f, "void"),
+            Self::Integer(it) => write!(f, "i{}", it.num_bits()),
```

Comment:
So if I have just the `IntegerType` I can't print it any more? I'd need to have it wrapped in `Type`?

Reply:
`IntegerType` still implements `Debug`.